rad2/src/commands/profiles.rs

```diff
@@ -115,6 +115,7 @@ impl Commands {
     }
 }
 
+// FIXME[ENTITY] resolve a User URN and return the User, not just a profile
```

Comment:
Yeah I think profile is obsolete anyway: you're supposed to have exactly one `User` unless you're using multiple keypairs, so we can just store the URN in some config.